includes/class-wc-order-item-product.php

```diff
@@ -167,7 +167,7 @@ public function set_taxes( $raw_tax_data ) {
 	 */
 	public function set_variation( $data ) {
 		foreach ( $data as $key => $value ) {
-			$this->add_meta_data( str_replace( 'attribute_', '', $key ), $value, true );
```

Comment:
Now you corrected me I'm not sure :) When they are stored in the backend (meta) do they have a prefix? If thats consistent with 2.6.x it should be fine.